apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java

```diff
@@ -16,40 +17,48 @@
 /**
  * @author alukin@gmail.com
  */
+@Slf4j
 class PeerLoaderThread implements Runnable {
     private static final Logger LOG = LoggerFactory.getLogger(PeerLoaderThread.class);
     private final List<String> defaultPeers;
     private final List<Future<String>> unresolvedPeers;
     private final Set<PeerDb.Entry> entries = new HashSet<>();
     private final TimeService timeService;
     private final PeersService peersService;
-    private PeerDb peerDb;
+    private final PeerDb peerDb;
 
-    public PeerLoaderThread(List<String> defaultPeers, List<Future<String>> unresolvedPeers, TimeService timeService, PeersService peersService) {
+    public PeerLoaderThread(List<String> defaultPeers,
+                            List<Future<String>> unresolvedPeers,
+                            TimeService timeService,
+                            PeersService peersService) {
         this.defaultPeers = defaultPeers;
         this.unresolvedPeers = unresolvedPeers;
         this.timeService = timeService;
         this.peersService = peersService;
+        this.peerDb = peersService.getPeerDb();
```

Comment:
Maybe better to inject PeerDb?

Reply:
That was first variant, but I changed that later to less params in construcor.